in_toto/verifylib.py

```diff
@@ -984,10 +985,8 @@ def verify_threshold_constraints(layout, chain_link_dict):
     reference_keyid = key_link_dict.keys()[0]
     reference_link = key_link_dict[reference_key]
 
-    # Iterate over all links to check their signature and
-    # compare their properties with a reference_link
+    # Iterate over all links to compare their properties with a reference_link
     for keyid, link in six.iteritems(key_link_dict):
-      keys_dict = {keyid: layout.keys.get(keyid)}
```

Comment:
Hmm, this change is confusing to me. Can you elaborate on what is this doing exactly?

Reply:
This change removes a variable that became obsolete with the removal of signature verification inside `verify_threshold_constraints` in #85 more specifically 3deb4bbb97e5f0828c855759030c83f191a145c4.

Reply:
Oh ok. Thanks for answering!